ffi/java-idsdk/README.md

```diff
@@ -0,0 +1,69 @@
+# Java wrapper for the ID SDK
+
+FFI wrapper using JNA
+
+## Build
+
+`mvn pakage`
+
+## Test
+
+A sample app is in the test/java directory: `com.iotics.sdk.identity.App`
+
+## Use
+
+Initialise the API with:
+
+```java
+SdkApi api = new JnaSdkApiInitialiser("<path_to>/lib-iotics-id-sdk-amd64.so").get();
+```
+
+or, if your `so` library is installed at `./bin/lib-iotics-id-sdk.so`, use:
+
+```java
+SdkApi api = new JnaSdkApiInitialiser().get();
+```
+
+Usages:
```

Comment:
Rather than putting this in a readme - can't we include a `.java` file with the examples in it?